src/components/ClusterOverview/Details/Details.js

```diff
@@ -8,51 +9,57 @@ import {
   DashboardCardTitle,
 } from '../../Dashboard/DashboardCard';
 import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
+import { Loading } from '../../Loading';
```

Comment:
The loading component still has `blank-slate-pf` style inside. I would suggest to make two component `<Loading>` and `<InlineLoading>`